www/include/monitoring/downtime/listDowntime.php

```diff
@@ -179,18 +177,25 @@
         $centreon->user->access->getAccessGroupsString() . ") ";
 }
 $request .= (isset($search_service) && $search_service != "" ? "AND 1 = 0 " : "") .
```

Comment:
Is it possible to simplify or make these string concatenations more readable?

Reply:
I could, but not sure it is the purpose in this pull request